ios/CustomerioReactnative.swift

```diff
@@ -75,24 +76,6 @@ class CustomerioReactnative: NSObject {
         CustomerIO.shared.deviceAttributes = data
     }
     
-    /**
-     Configure properties like autoTrackDeviceAttributes, logLevel etc
-f     */
-    private func config(data : Dictionary<String, AnyHashable>) -> Void{
```

Comment:
Did we deliberately remove this? 👀

Reply:
We set config with [initialization](https://github.com/customerio/customerio-reactnative/blob/0acfdaea905618bb9e40899770b9c4c51299435e/ios/CustomerioReactnative.swift#L19) of the package so this function is no longer called. Hence removed

Reply:
Makes sense 👍🏻